HITDAD/js/plugins/SPFDadJoke.js

```diff
@@ -60,26 +60,26 @@
       }
   }
 
-  document.addEventListener("mousedown", function (event) {
-      // make sure not on phone or carrying a box and is left click
-      if ($dataMap && !$gamePlayer.isCarrying() && !SPF_OnPhone(event) && event.button === 0) {
-
-        var item = SPF_FindItemById(ITEM_ID);
-        console.log("Here we gooo!", item, SPF_IsItemSelected(item));
-
-        if (!SPF_isEmpty(item) &&
-             SPF_IsItemSelected(item) &&
-            !Input._isItemShortCut()) { // Do not fire if hotbar is open.
-
-          chargeAttack();
-        }
-
-      }
-  });
+  // document.addEventListener("mousedown", function (event) {
+  //     // make sure not on phone or carrying a box and is left click
+  //     if ($dataMap && !$gamePlayer.isCarrying() && !SPF_OnPhone(event) && event.button === 0 && false) {
+  //
+  //       var item = SPF_FindItemById(ITEM_ID);
+  //       console.log("Here we gooo!", item, SPF_IsItemSelected(item));
+  //
+  //       if (!SPF_isEmpty(item) &&
+  //            SPF_IsItemSelected(item) &&
+  //           !Input._isItemShortCut()) { // Do not fire if hotbar is open.
+  //
+  //         chargeAttack();
+  //       }
+  //
+  //     }
+  // });
 
   document.addEventListener("mouseup", function (event) {
-
-      if ($dataMap) {
+      // Only call if this item is equipped
+      if ($dataMap && SPF_CSI.id === ITEM_ID) {
```

Comment:
```suggestion
      if ($dataMap &&  SPF_CSI && SPF_CSI.id === ITEM_ID) {
```
This is ensures we check that SPF_CSI is not null before getting ID from it. Otherwise, we get a crash on the cutscene.

Reply:
Looking through the rest of your code, there may be a couple places where SPF_CSI is not initialized and we'll get an error if we try to get SPF_CSI.id so that's something to watch out for. The best way to check if not null is just do:
 if (SPF_CSI) { ... }